### PR TITLE
Revert "Revert "Feature/devops""

### PR DESCRIPTION
**Pull request by monodelic**
This reverts commit 357f633b4a12f20bbd84484a68a4e772b5d273fe.

### CODE REVIEW

File: newrelic.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Generated March 14, 2019
+#
+# This configuration file is custom generated for Home_3660
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: e833ffa37df9d5af56540553d736145a9299c1bd
```
Comment (artempartos) [inline newrelic.yml]: do it through env var and add to docker-compose

File: config/initializers/rollbar.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = 'b47585744d104ac283829af936fdc67e'
```
Comment (artempartos) [inline config/initializers/rollbar.rb]: do it through env var and add to docker-compose

File: .coveralls.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+repo_token: khcaxtwPaTT1iUbcptGAPag8uqO2lWPFf
```
Comment (artempartos) [inline .coveralls.yml]: do it through env var and add to docker-compose

File: test/test_helper.rb [hunk 4]
```diff
@@ -1,6 +1,13 @@
+require 'coveralls'
+Coveralls.wear!('rails')
```
Comment (artempartos) [inline test/test_helper.rb:2]: add coveralls badge to readme

**Review by artempartos — CHANGES_REQUESTED** [review]

File: newrelic.yml (hunk 1 above)
Comment (monodelic) [reply newrelic.yml]: ✔️

File: config/initializers/rollbar.rb (hunk 2 above)
Comment (monodelic) [reply config/initializers/rollbar.rb]: ✔️

File: .coveralls.yml (hunk 3 above)
Comment (monodelic) [reply .coveralls.yml]: ✔️

File: test/test_helper.rb (hunk 4 above)
Comment (monodelic) [reply test/test_helper.rb:2]: ✔️

**Comment by artempartos** [comment]
rebase it and solve conflicts

**Comment by monodelic** [comment]
✔️

**Comment by artempartos** [comment]
👍

**Pull request merged**